mod_form.php

```diff
@@ -187,10 +187,10 @@ public function definition() {
             $unissuedheader = false;
             foreach ($usersearnedcertificate as $user) {
                 $existingcertificate = false;
-
+                $useremail = strtolower($user->email);
```

Comment:
I'd name it as `$lowercaseemail` or a similar name because `$useremail` doesn't tell a difference from `$user->email`. I also wouldn't mind not defining the variable as `if ($certificate->recipient->email == strtolower($user->email))`.

Reply:
I have removed the variable definition as you suggested for both methods